middlewares/hacker.middleware.js

```diff
@@ -492,6 +492,18 @@ async function updateHacker(req, res, next) {
     }
 }
 
+/**
+ * Sets Hacker Status to Accepted and runs it through updateHacker's functionality.
```

Comment:
Nit: Rename function to parseAccept, similar to parseConfirmation and make comment just say it sets the req.body.status to accepted